crowbar_framework/app/helpers/sanities_helper.rb

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2016, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module SanitiesHelper
+  def flash_for_sanity_checks(errors)
+    html = "<h3>#{I18n.t("sanities.show.header")}</h3>"
```

Comment:
:-1: for this. If you write content structure within helpers just use the `content_tag` helpers.